index.html

```diff
@@ -178,5 +178,35 @@ <h2 class="lang-title">Skills</h2>
       </div>
     </div>
   </section>
+
+  <!-- Contact form section as per required in https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md -->
+
+  <section class="form">
+    <h2 class="formTitle">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form action="https://formspree.io/f/mzbokrka" method="post">
+      <ul class="formUl">
+        <li  class="formInput">
+          <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+        </li>
+        <li class="formInput">
+          <input type="email" id="mail" name="email" placeholder="E-mail address" required>
+        </li>
+        <li class="formInput" id="no-border">
+          <textarea  class="textarea1" name="message" maxlength="500" required placeholder="Enter text here"></textarea>
+        </li>
+      </ul>
+      <button type="submit" class="main-button">Get in touch</button>
+    </form>
+    <div id="marginTop">
+      <ul class="iconsBot">
+        <li><img src="./img/social/dark-icons/Github.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Linkedin.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Angelist.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Twitter.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Medium.png" alt=""></li>
+      </ul>
```

Comment:
- [ ] [OPTIONAL] these are links to your social media so you should add an `<a>`  tag to all and your social media URL